### ISSUE TITLE
How to use LoadingObjectManager? Are there any solutions?

### ISSUE THREAD

**Issue by nadador2011** [gribnoysup/react-yandex-maps#116]
Hi!

react-yandex-maps has ObjectManager component for managment of object collections, but I need dinamically load objects with LoadingObjectManager in yandex API.  

Are there any solutions for this with the current version of react-yandex-maps or do I need to create wrapper component for LoadingObjectManager as an ObjectManager?

**Comment by mmarkelov** [comment]
@nadador2011 Hi! There is now solution for **LoadingObjectManager**, you can use hoc **withYMaps**. You can check it [here](https://react-yandex-maps.now.sh/provider-consumer/with-ymaps-hoc). I need time to figure out on implementation, cause I just haven't got any experience with it.

**Comment by nadador2011** [comment]
@mmarkelov Yes, when I use hoc withYMaps I can use **LoadingObjectManager**
`const objectManager = new ymaps.LoadingObjectManager('https://myServer.com/tile?bbox=%b, {})`
as described in the yandex API documentation, but after that it's necessarry add **objectManager** to the map `myMap.geoObjects.add(objectManager)`.
In the react-yandex-maps it's implemented with hoc **withParentContext**, but it's inner component of react-yandex-maps.

**ObjectManager**, **LoadingObjectManager**, **RemoteObjectManager** extends the same objects and has the same way of adding them to map  `myMap.geoObjects.add(objectManager)`, therefore I think it would be great, if react-yandex-maps would have component for adding objectManager to the map, instead of separate components for ObjectManager(already implemented), LoadingObjectManager and RemoteObjectManager.

**Comment by mmarkelov** [comment]
@nadador2011 I just started work on it. So you can check [my branch](https://github.com/mmarkelov/react-yandex-maps/tree/Implement_ObjectManager_extensions)
It will be very helpful if you could provide some example of **LoadingObjectManager** cause I just don't have any experience with it and it is a bit difficult to work with it
